iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesParser.java

```diff
@@ -78,6 +78,7 @@ private FileTree parseHelmFile(String source, @Nullable InputFileContext inputFi
   }
 
   private FileTree evaluateAndParseHelmFile(String source, InputFileContext inputFileContext) {
+    readLinesSizes(source, inputFileContext, locationShifter);
     var fileRelativePath = getFileRelativePath(inputFileContext);
     var evaluatedSource = helmProcessor.processHelmTemplate(fileRelativePath, source, inputFileContext);
     var evaluatedAndCleanedSource = cleanSource(evaluatedSource, inputFileContext, locationShifter);
```

Comment:
Actually, not related to this PR per se, but why not call `cleanSource` inside `HelmProcessor#processHelmTemplate`?

Reply:
Separation of responsibility. The `HelmProcessor` already does a lot and returns the evaluated template directly from Go executable. It's the exact text we can produce using `helm -s templates/pod.yaml .` I would like to have it for easier debugging purposes. 
But I'm open to discussion and maybe introducing something between `KubernetesParser` and `HelmProcessor` is needed. 🤔 And what is a good name for it?

Reply:
Maybe `HelmParser` is a good name?

Reply:
I'd rather call it `HelmPostProcessor` and call inside `HelmProcessor` :D Then existing logic can be encapsulated in `HelmPreProcessor`, so that `HelmProcessor` simply calls `HelmPreprocessor`, `HelmEvaluator` and then `HelmPostprocessor`. But it's not a final thought still.
I think `HelmParser` would be a confusing name as it doesn't parse and is in fact part of the `KubernetesParser`

Reply:
Ticket created: https://sonarsource.atlassian.net/browse/SONARIAC-1289